Makefile

```diff
@@ -5,6 +5,7 @@ unit_test:
 
 integration_test:
 	docker-compose run --rm processing bash -c "cd data_collection && scrapy check"
+	docker-compose run --rm processing black . --check
```

Comment:
If the intention is to run this command in the continuous integration, it should be placed in the `travis.yml` file. Travis CI actually [runs](https://github.com/okfn-brasil/diario-oficial/blob/3741d244f0496bcacb4d6b1d8259917c65a9cc73/.travis.yml#L16-L17) `make unit_test` after setup.

Reply:
Hum… ok. Semantically it is not an integration test. I just wanted to find a way to offer a kind of shortcut for local checks. Should we add another command to `Makefile` in this case (in addition to adding this command to `.travis.yml`)?

Reply:
Since you added pre-commit installation to the Makefile, you could create a new `make ci` command to run the existing script section plus black's check.

Reply:
> Since you added pre-commit installation to the Makefile, you could create a new `make ci` command to run the existing script section plus black's check.

Sounds good! Gonna register this on an issue and get back to it one of these days : )